app/tests/settings.py

```diff
@@ -10,9 +10,15 @@
 from config.settings import *  # noqa: F401, F403, E402, I251
 
 SESSION_COOKIE_DOMAIN = ".testserver"
-ALLOWED_HOSTS = [SESSION_COOKIE_DOMAIN]
+ALLOWED_HOSTS = [SESSION_COOKIE_DOMAIN, ".localhost"]
 SECURE_SSL_REDIRECT = False
 
+TEST_TEMPLATE_DIR = os.path.join(
+    SITE_ROOT,  # noqa 405
```

Comment:
We can use pathlib in all of the (non-third-party) Django settings since Django 3.1. Might be nicer to update `SITE_ROOT` to use that.